native/android/app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <color name="accent">@color/bpkBlue500</color>
+
+    <!--default background on API level 26 onwards -->
+    <color name="background">#FAFAFA</color>
```

Comment:
This should use a backpack token too right? Also whats the requirement for this file? It just redefines backpack tokens under a new alias, cant we just use them directly in `native/android/app/src/main/res/values/styles.xml` ?

Reply:
tokens do not have a value for `#FAFAFA` which is the default window background on new Android versions. Also `styles.xml` contains styles and `colors.xml` contains only color values. Just an Android convention.

Reply:
> tokens do not have a value for #FAFAFA which is the default window background on new Android versions.

We want to use a backpack token for this though :) I suggest `colorWhite`.

> Also styles.xml contains styles and colors.xml contains only color values. Just an Android convention.

If we are just consuming backpack colours though, we don't need `colors.xml` do we? We just consume the colours from backpack directly in `styles.xml`? Or is it common to redeclare resources from dependencies?

Reply:
> We want to use a backpack token for this though :) I suggest colorWhite.

Bear in mind that if we *do* use something other than #fafafa, we'll need to retake every Android screenshot.

Not saying we should, just something to be aware of as it'll add scope.

Reply:
I'm happy to do it in another PR to chunk up the blast radius.

In almost all cases we should be using a backpack token. 🎒🙌